Integrations/Integration-Wildfire.yml

```diff
@@ -0,0 +1,261 @@
+configuration:
+- display: Server URL (e.g. https://192.168.0.1)
+  name: server
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: API Key
+  name: token
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: secure
```

Comment:
Field means to skip validation which is nor secure, so a proper name should be insecure (as when it's true, you are not secured)